pkg/kubelet/cm/devicemanager/manager.go

```diff
@@ -669,7 +669,7 @@ func (m *ManagerImpl) allocateContainerResources(pod *v1.Pod, container *v1.Cont
 		// and container2 both require X resource A and Y resource B. Both allocation
 		// requests may fail if we serve them in mixed order.
 		// TODO: may revisit this part later if we see inefficient resource allocation
-		// in real use as the result of this. Should also consider to parallize device
+		// in real use as the result of this. Should also consider to paralyze device
```

Comment:
typo here, should be parallelize

Reply:
oh, yes, I have corrected it, thanks for reminder.